pennylane/workflow/get_best_diff_method.py

```diff
@@ -29,18 +30,9 @@ def get_best_diff_method(qnode: QNode):
     """Returns a function that computes the 'best' differentiation method
     for a particular QNode.
 
-    This method prioritizes differentiation methods in the following order (SPSA-based and Hadamard-based gradients
-    are not included here):
-
-    * ``"device"``
-    * ``"backprop"``
-    * ``"parameter-shift"``
-
-    .. note::
-
-        The first differentiation method that is supported (from top to bottom)
-        will be returned. The order is designed to maximize efficiency, generality,
-        and stability.
```

Comment:
This description should be kept. Yes, it is describing logic in `_resolve_execution_config`, but that is more developer facing, and this is more user facing.

Reply:
reverted https://github.com/PennyLaneAI/pennylane/pull/7898/commits/1efb81a6be74865d675fa06418b22f8e7582182d